.github/workflows/ansible-test.yml

```diff
@@ -364,6 +368,13 @@ jobs:
         # sometimes this hangs forever waiting for an IP
         uses: ./.github/actions/macos-docker
 
+      - name: cgroups, see groups run, run groups run (please run!)
+        if: ${{ startsWith(matrix.runner, 'macos') }}
+        run: |
```

Comment:
This appears to be running on the macOS host instead of the Docker host.

Reply:
yeah... I'm not sure how I would run that in the docker VM 🤔